local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -58,30 +59,38 @@ func NewVolumeUtil() VolumeUtil {
 }
 
 // IsDir checks if the given path is a directory
-func (u *volumeUtil) IsDir(fullPath string) (bool, error) {
+func (u *volumeUtil) IsDir(fullPath string) error {
```

Comment:
Had a longer offline chat with Ian about this. My concern was that the function returns err in times when a boolean was preferable, so going with one return type seemed simpler. Having  said that, keeping in mind the k8s standards, I will change it back to its original signature.